connexion/api.py

```diff
@@ -164,7 +171,8 @@ def add_paths(self, paths=None):
                         import sys
                         logger.error(error_msg)
                         et, ei, tb = sys.exc_info()
-                        raise ei.with_traceback(tb)
+                        ei.__traceback__ = tb
```

Comment:
TL;TR: you should undo this change and let your PR's focused in every change you want to make.